app/models/student_attendance.rb

```diff
@@ -0,0 +1,7 @@
+class StudentAttendance < ApplicationRecord
+  belongs_to :user
```

Comment:
Since we have to go back and fix the migration commented on above, let's make one more tweak. Let's rename the `user_id` column on `student_attendance` to `student_id`.

This association would then be defined like this:
```ruby
  belongs_to :student, class_name: "User", foreign_key: :student_id
```